src/System.Device.Gpio/System/Device/Gpio/GpioController.cs

```diff
@@ -534,7 +535,7 @@ private static GpioDriver GetBestDriverForBoardOnLinux()
                 // For now, for Raspberry Pi 5, we'll use the LibGpiodDriver.
                 // We need to create a new driver for the Raspberry Pi 5,
                 // because the Raspberry Pi 5 uses an entirely different GPIO controller (RP1)
-                return new LibGpiodDriver(4);
+                return new LibGpiodDriver(4); // TODO: Use chip selection
```

Comment:
Two things:
- what are the steps to implement the "TODO"?
- I would place a `const int` instead of a literal in order to explain the meaning of that

Reply:
Fixed